sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/TryCastSuite.scala

```diff
@@ -48,4 +48,8 @@ class TryCastSuite extends AnsiCastSuiteBase {
   override def checkCastToNumericError(l: Literal, to: DataType, tryCastResult: Any): Unit = {
     checkEvaluation(cast(l, to), tryCastResult, InternalRow(l.value))
   }
+
+  test("try_cast: to_string") {
+    assert(TryCast(Literal("1"), IntegerType).toString == "try_cast(1 as int)")
+  }
```

Comment:
Test `sql` as well?

Reply:
It's already verified in the `try_cast.sql.out`.